### PR TITLE
Ap 3487 additional merits q3

### PR DESCRIPTION
**Pull request by kmahern**
## What

[Link to story](https://dsdmoj.atlassian.net/jira/software/c/projects/AP/boards/233?modal=detail&selectedIssue=AP-3487)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

File: app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module Providers
+  module ApplicationMeritsTask
+    class NatureOfUrgencyController < ProviderBaseController
```
Comment (cpjmcquillan) [inline app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb:3]: I've mentioned this (i.e pluralisation) a couple times across various PRs.

I definitely think it's worthwhile defaulting to convention (e.g NatureOfUrgencies) unless there is a clear reason not to.

How do the rest of the team feel about this?

Comment (cpjmcquillan) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb:3]: I'd even suggest this could just be an Urgencies controller!

Comment (jsugarman) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb]: i had started with a plural controller name but changed to singular as the route and relations are all singular and convention only [favours pluralisation](https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention) but does not adhere to it itself (i.e. ApplicationController).

Comment (kmahern) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb]: I have renamed the controller (+route etc.) urgencies_controller.rb. @colinbruce @jsugarman @cpjmcquillan are we all happy with the new naming?

Comment (jsugarman) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb]: I would have said its fine to make it plural if its already done as it is not something i think we need to sweat over too much, but `urgencies` for `nature_of_urgenc[ies|y]` i have to say I am not keen on

Comment (cpjmcquillan) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb]: I meant to delete my `urgencies` comment 😆 

I think `NatureOfUrgenciesController` makes most sense in the context of using a `NatureOfUrgency` form model to create `Urgency` records.

Comment (kmahern) [reply app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb]: 👍 ok, I have named the controller NatureOfUrgenciesController

**Pull request merged**